unlock-js/src/__tests__/helpers/fixtures/locks.js

```diff
@@ -10,13 +10,13 @@ export default {
       name: null, // Not supported in v0
       currencyContractAddress: null, // Not supported in v0
     },
-    {
-      expirationDuration: 60 * 60 * 24 * 30,
-      keyPrice: '0.1',
-      maxNumberOfKeys: UNLIMITED_KEYS_COUNT,
-      name: null, // Not supported in v0
-      currencyContractAddress: null, // Not supported in v0
-    },
+    // {
```

Comment:
Should this be deleted instead of commented?

Reply:
it's a mistake it should be neither commented nor deleted! Adding it back now.